autologout.module

```diff
@@ -472,3 +481,46 @@ function autologout_user_login($account) {
       ->execute();
   }
 }
+/**
+ * Implements hook_page_attachments().
+ */
+function autologout_page_attachments(array &$page) {
+
+  $refresh_only = _autologout_refresh_only();
+  $now = time();
```

Comment:
Use [`REQUEST_TIME`](https://api.drupal.org/api/drupal/core%21includes%21bootstrap.inc/constant/REQUEST_TIME/8) instead of the `time()` function.